<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CQDlgSelection</class>
 <widget class="QDialog" name="CQDlgSelection">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>430</width>
    <height>255</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Selection</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>411</width>
     <height>191</height>
    </rect>
   </property>
   <property name="title">
    <string>Object selection</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <widget class="QToolButton" name="simpleShapesButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>25</y>
      <width>190</width>
      <height>18</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QToolButton" name="groupedShapesButton">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>25</y>
      <width>190</width>
      <height>18</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QToolButton" name="millsButton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>145</y>
      <width>190</width>
      <height>18</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QToolButton" name="dummiesButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>45</y>
      <width>190</width>
      <height>18</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QToolButton" name="lightsButton">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>45</y>
      <width>190</width>
      <height>18</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QToolButton" name="camerasButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>65</y>
      <width>190</width>
      <height>18</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QToolButton" name="proximitySensorsButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>85</y>
      <width>190</width>
      <height>18</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QToolButton" name="renderingSensorsButton">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>65</y>
      <width>190</width>
      <height>18</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QToolButton" name="forceSensorsButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>105</y>
      <width>190</width>
      <height>18</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QToolButton" name="jointsButton">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>85</y>
      <width>190</width>
      <height>18</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QToolButton" name="graphsButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>125</y>
      <width>190</width>
      <height>18</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QToolButton" name="pathsButton">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>105</y>
      <width>190</width>
      <height>18</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QToolButton" name="mirrorsButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>165</y>
      <width>190</width>
      <height>18</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QToolButton" name="octreesButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>145</y>
      <width>190</width>
      <height>18</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QToolButton" name="pointCloudsButton">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>125</y>
      <width>190</width>
      <height>18</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
  </widget>
  <widget class="QToolButton" name="clearSelectionButton">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>210</y>
     <width>201</width>
     <height>18</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string>Clear selection</string>
   </property>
  </widget>
  <widget class="QToolButton" name="invertSelectionButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>210</y>
     <width>201</width>
     <height>18</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string>Invert selection</string>
   </property>
  </widget>
  <widget class="QLabel" name="selectionInfo">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>411</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>simpleShapesButton</tabstop>
  <tabstop>groupedShapesButton</tabstop>
  <tabstop>dummiesButton</tabstop>
  <tabstop>lightsButton</tabstop>
  <tabstop>camerasButton</tabstop>
  <tabstop>renderingSensorsButton</tabstop>
  <tabstop>proximitySensorsButton</tabstop>
  <tabstop>jointsButton</tabstop>
  <tabstop>forceSensorsButton</tabstop>
  <tabstop>pathsButton</tabstop>
  <tabstop>graphsButton</tabstop>
  <tabstop>pointCloudsButton</tabstop>
  <tabstop>octreesButton</tabstop>
  <tabstop>millsButton</tabstop>
  <tabstop>mirrorsButton</tabstop>
  <tabstop>clearSelectionButton</tabstop>
  <tabstop>invertSelectionButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
